lib/things.py

```diff
@@ -11,13 +11,12 @@ def __init__(self, url):
     def create(self, thing, token):
         '''Creates thing entity in the database'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/things", json=thing, headers={"Authorization": token})
```

Comment:
Same as [here](https://github.com/mainflux/sdk-py/pull/6/files#r780405555)

Reply:
Remove the `/things/bulk`